src/main/cluster.ts

```diff
@@ -76,6 +76,7 @@ export class Cluster implements ClusterModel, ClusterState {
    * If extension sets this it needs to also mark cluster as enabled on activate (or when added to a store)
    */
   public ownerRef: string;
+  public initializing = false;
```

Comment:
Probably should be observable field since participating in `autorun` in cluster-manager.

Reply:
Changed as observable.